### DISCUSSION TITLE
How can I set a texture on a placed playerhead(as a Block)?

### DISCUSSION THREAD

**Discussion by FloGamerMC** [Minestom/Minestom#752] [Help] [answered]
I would like to set a texture on a placed playerhead. Or is there a other way to load textures from Playerheads in a saved Anvil World.

**Comment by emortaldev** [comment, marked as answer]
It is in kotlin, but you can use a [block handler](https://github.com/EmortalMC/Immortal/blob/main/src/main/kotlin/dev/emortal/immortal/blockhandler/SkullHandler.kt) to read it straight from the anvil world (this is not how you would do it programmatically with `instance.setBlock(...)` however)

**Reply by FloGamerMC** [reply]
It is in Java but I translate it und now it works. Thank you
